src/server/game/Battlegrounds/Zones/BattlegroundRV.h

```diff
@@ -77,15 +77,16 @@ enum BattlegroundRVData
     BG_RV_STATE_SWITCH_PILLARS,
     BG_RV_STATE_CLOSE_FIRE,
 
-    BG_RV_PILLAR_SWITCH_TIMER                    = 25000,
-    BG_RV_FIRE_TO_PILLAR_TIMER                   = 20000,
-    BG_RV_CLOSE_FIRE_TIMER                       =  5000,
-    BG_RV_FIRST_TIMER                            = 20500, // elevators rise in 20133ms
     BG_RV_WORLD_STATE_A                          = 0xe11,
     BG_RV_WORLD_STATE_H                          = 0xe10,
     BG_RV_WORLD_STATE                            = 0xe1a,
 };
 
+static constexpr Milliseconds BG_RV_PILLAR_SWITCH_TIMER  = 25s;
+static constexpr Milliseconds BG_RV_FIRE_TO_PILLAR_TIMER = 20s;
+static constexpr Milliseconds BG_RV_CLOSE_FIRE_TIMER     = 5s;
+static constexpr Milliseconds BG_RV_FIRST_TIMER          = 20500ms; // elevators rise in 20133ms
```

Comment:
maybe move this in source file `.cpp` ?

Reply:
I can do it, just want to learn why?

Reply:
I think the less information in the headers, the better

Reply:
done